src/packages/error/error-string.ts

```diff
@@ -10,6 +10,7 @@ export const ERROR_STRINGS = {
     [PackageErrorCode.BlockstackIdNamespaceValidation]: "Invalid Blockstack ID",
     [PackageErrorCode.BnsEmptyData]: "No name data available",
     [PackageErrorCode.BnsResolutionFailed]: "'{0}' node not available because '{1}'",
+    [PackageErrorCode.ClientNotInitialized]: "Please initialize the client before calling getAssetMapping",
```

Comment:
is this error specific to `getAssetMapping`? Then you should make the error name specific.